tests/crud/test_session.py

```
@@ -9,6 +12,164 @@
+            models.SessionPeer.peer_name == test_peer.name,
+            models.SessionPeer.workspace_name == test_workspace.name,
+        )
+
+        await crud.set_peers_for_session(
+            db_session,
+            workspace_name=test_workspace.name,
+            session_name=session_name,
+            peer_names={test_peer.name: original_config},
+        )
+        first_result = await db_session.execute(session_peer_stmt)
+        first_joined_at, first_left_at, first_config = first_result.one()
```

Ruff (0.16.1)

[warning] Unpacked variable `first_joined_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)